org.aegisub.Aegisub.yml

```diff
@@ -0,0 +1,171 @@
+
+    - name: luajson
+      buildsystem: simple
+      sources:
+          - type: git
+            url: https://github.com/harningt/luajson.git
+      build-commands:
+          - mkdir -p /app/share/aegisub/automation/include/
+          - cp -r lua/* /app/share/aegisub/automation/include/
+
+    - name: ffi-experiments
+      buildsystem: simple
+      sources:
+          - type: git
+            url: https://github.com/TypesettingTools/ffi-experiments.git
+      build-commands:
+          - luarocks install moonscript
+          - make -B lua
+          #  - luarocks remove moonscript # TODO: remove again, to not end in the final pack, or is it still needed?
+          - make -B all
+          - mkdir -p /app/share/aegisub/automation/include/requireffi/
+          - cp build/requireffi.lua /app/share/aegisub/automation/include/requireffi/
+          - mkdir -p /app/share/aegisub/automation/include/BM/
+          - cp build/BadMutex.lua /app/share/aegisub/automation/include/BM/
+          - mkdir -p /app/share/aegisub/automation/include/DM/
+          - cp build/DownloadManager.lua /app/share/aegisub/automation/include/DM/
+          - mkdir -p /app/share/aegisub/automation/include/PT/
+          - cp build/PreciseTimer.lua /app/share/aegisub/automation/include/PT/
+          - cp build/lib*.so /app/lib/
+      build-options:
+          build-args:
+              - --share=network
+
+    - name: DependencyControl
+      buildsystem: simple
+      sources:
+          - type: archive
+            url: https://github.com/TypesettingTools/DependencyControl/archive/refs/tags/v0.6.4-alpha.tar.gz
+            sha256: b4ba3da034be85d2394d4ec7de4192a76be45f900a60dd484a0d1af3d6671c71
+      build-commands:
+          - mkdir -p /app/share/aegisub/automation/autoload/
+          - cp macros/l0.DependencyControl.Toolbox.moon /app/share/aegisub/automation/autoload/
+          - mkdir -p /app/share/aegisub/automation/include/l0/
+          - cp -r modules/* /app/share/aegisub/automation/include/l0/
+
+    - name: boost
+      buildsystem: simple
+      sources:
+          - type: archive
+            url: https://boostorg.jfrog.io/artifactory/main/release/1.83.0/source/boost_1_83_0.tar.gz
+            sha256: c0685b68dd44cc46574cce86c4e17c0f611b15e195be9848dfd0769a0a207628
+      build-commands:
+          - ./bootstrap.sh
+          - ./b2 --prefix=/app --with-chrono --with-thread --with-locale --with-regex --with-system install variant=release
+
+    - name: libunibreak
+      buildsystem: simple
+      sources:
+          - type: archive
+            url: https://github.com/adah1972/libunibreak/releases/download/libunibreak_5_1/libunibreak-5.1.tar.gz
+            sha256: dd1a92d4c5646aa0e457ff41d89812ec5243863be6c20bbcb5ee380f3dd78377
+      build-commands:
+          - mkdir build
+          - cd build && ../configure --prefix=/app
+          - cd build && make
+          - cd build && make install
```

Comment:
libunibrak is only used for an API-toggled ASS-incompatible mode not used by Aegisub, so including it here is not needed.

Reply:
Resolved by https://github.com/TypesettingTools/Aegisub/pull/189/commits/2bf43aa30f3c7b2d252d08de7d7b91c075b121b5